src/request/components/RequestDetailPanelRequest.tsx

```diff
@@ -14,7 +14,7 @@ export interface IRequestProps {
     };
     response: {
         body: string;
-        headers: { [key: string]: string };
+        headers: { [key: string]: string | string[] };
```

Comment:
Do we think its worth creating a type for `HeaderValues`? Might be easier than having to make sure that we repeat these semantics through the code.